Releasing.md

```diff
@@ -312,7 +312,7 @@ It's best practice to use the automation script (mentioned in the release templa
 
 ### When
 
-On Mondays, one week before main apps (WPiOS & WPAndroid) have cut their releases, every 2 weeks.
+On Fridays, the week before the main apps (WPiOS & WPAndroid) have cut their releases, every 2 weeks.
```

Comment:
I know that we have cut ( and/or finalized) the release on Fridays but since the schedule states that we start on Thursday is it best to put Thursday here instead? Let me know.
![release-schedule](https://user-images.githubusercontent.com/1509205/134959359-51e1156f-d5df-4711-ad5c-65af7d491a77.png)

Reply:
I guess it's fine since the process is started on Thursday and the final integration is done Friday. 
This part of the documentation below says we create and integrate on Friday but it would seem like we _create_ on Thursday and _integrate_ on Friday. Wdyt?
<img width="1361" alt="Screenshot 2021-09-27 at 12 49 42 PM" src="https://user-images.githubusercontent.com/1509205/134959605-79bcecf8-3df5-4a9f-b2f3-c8ff27883d7c.png">

Reply:
Sounds good! I wasn't sure if it was Thursday or Friday, but it sounds like Thursday closer matches our processes.